db/seeds.rb

```diff
@@ -139,18 +139,32 @@
 trip17.save
 
 image18 = File.open('app/assets/images/tianmen.jpg')
-trip18 = Trip.new(destination_id: 1,activity_id: 3,name: "Tianmen Mountains",
+trip18 = Trip.new(destination_id: 4, activity_id: 5, name: "Tianmen Mountains",
                   description: "The only thing more exhilarating than the climb to the top is the fall to the bottom in this remote Chinese valley.",
                   trip_cost: 28550.00, lat: 29.0527, long: 110.4830)
 trip18.image = image18
 trip18.save
 
-
-Extra.create(name: "Monkey Banana-Eating Contest", description: "description", cost: 2000.00)
-Extra.create(name: "Shark Slalom Surfing", description: "description", cost: 4000.00)
-Extra.create(name: "Barometer Free Skydiving", description: "description", cost: 3000.00)
-Extra.create(name: "Rhino Jousting", description: "description", cost: 8000.00)
-Extra.create(name: "Giraffe Racing", description: "description", cost: 6000.00)
+image19 = File.open('app/assets/images/peru.jpg')
+trip19 = Trip.new(destination_id: 2, activity_id: 4, name: "The Cordillera Blanca",
+                  description: "Climb what some consider to be the most beautiful mountain range in the world.",
+                  trip_cost: 11000.00, lat: -9.0717, long: -77.3632)
+trip19.image = image19
+trip19.save
+
+image20 = File.open('app/assets/images/siberia.jpg')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.